contents/docs/llm-analytics/helicone-posthog.md

```diff
@@ -1,37 +1,16 @@
 ---
-title: LLM insights (beta)
+title: Integrating with Helicone
 availability:
     free: full
     selfServe: full
     enterprise: full
 ---
 
-We've teamed up with various LLM platforms to track metrics for LLM apps. This makes it easy to answer questions like:
+> LLM analytics is currently an [opt-in public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
```

Comment:
AFAIK, it isn't early access, but opt-in on Helicone's side.

Reply:
I'll defer to Lior to change this, but I think it's simplest if we're consistent around what we say our view of this is. Currently, everywhere else positions LLM analytics as a beta. But we can change this!

Reply:
@joethreepwood let's remove this since the opt in expereince is not built into early-access, so we're directing users to place which is not actionable to this
```suggestion
> LLM analytics is currently in public beta. This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
```